sike/sike.go

```diff
@@ -0,0 +1,33 @@
+// Package sike contains all the required interfaces for running an instance of
+// Supersingular Isogeny Key Encapsulation (SIKE) mechanism between two parties.
+// This package provide all the required APIs and generic data-types for SIKE 
```

Comment:
Can you rephrase it to that:
"""
This package contains implementation of Supersingular Isogeny Key Encapsulation (SIKE) protocol. SIKE is built on top of SIDH and implements KEM encapsulation and KEM decapsulation. 
Both, data types and API. provided by this package are generic and can be instantiated with specific parameters (see sikep751.go).
Author: Amir Jalali		ajalali2016@fau.edu
"""